pkgs/applications/misc/sigal/default.nix

```
@@ -1,14 +1,16 @@
-{ lib, python3Packages, ffmpeg }:
+{ lib, python3Packages, pythonAtLeast, ffmpeg }:
 
 python3Packages.buildPythonApplication rec {
-  version = "2.0";
+  version = "2.1";
   pname   = "sigal";
 
   src = python3Packages.fetchPypi {
     inherit version pname;
-    sha256 = "0ff8hpihbd30xjy155ksfpypjskilqg4zmyavgvpri8jaf1qpv89";
+    sha256 = "1ddjijpdg0hk8xzqsgnxicivscyh9lz9cxikkxi8yr9hw2wgwjnm";
   };
 
+  disabled = pythonAtLeast "3.6.0";
```

IIUC you don't need to add `pythonAtLeast` to the arguments:
```suggestion
  disabled = python3Packages.pythonAtLeast "3.6.0";
```